core/arch/arm/plat-d06/conf.mk

```diff
@@ -8,6 +8,7 @@ CFG_WITH_PAGER ?= n
 CFG_WITH_SOFTWARE_PRNG ?= y
 CFG_WITH_STATS ?= y
 CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= y
+CFG_HISILICON_CRYPTO_DRIVER ?= y
```

Comment:
https://github.com/OP-TEE/optee_os/pull/6217 does the same change in _conf.mk_. Sync with @loubaihui on which P-R gets merged first.

Reply:
Thank you for your tip. I'll pay attention.

Reply:
done

Reply:
#6217 has been merged.
Please rebase on top of master branch tip and remove this line.

Reply:
![image](https://github.com/OP-TEE/optee_os/assets/33485704/f2c695ab-0480-444d-b7d0-b00c0685fb4f)

Done, I moved the code from line 31 to line 11.